packages/components/src/reaction_icon/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bravoIcon from './images/bravo.svg';
+import goodIdeaIcon from './images/good_idea.svg';
+import loveIcon from './images/love.svg';
+import thankYouIcon from './images/thankyou.svg';
+import { REACTION_BRAVO, REACTION_GOOD_IDEA, REACTION_LOVE, REACTION_THANK_YOU } from '../constants';
+
+// TODO: move final icons to @goodhood/icons
+const ICONS = {
+  [REACTION_THANK_YOU]: thankYouIcon,
+  [REACTION_GOOD_IDEA]: goodIdeaIcon,
+  [REACTION_BRAVO]: bravoIcon,
+  [REACTION_LOVE]: loveIcon,
+};
+
+const ReactionIcon = ({ reaction, ...cleanProps }) => (
```

Comment:
1. Wanted to spare us from using svgr in the package. It's not quite easy because we run the code one time through webpack (storybook) and one time through rollup. We did the same thing in maps https://github.com/goodhood-eu/goodhood/blob/3d5942e4b60054db5790ed3e7ff36b0131b95990/packages/map/src/pin_marker/index.jsx#L9
2. We need the reactions-string-to-icon-conversion multiple times in the app. That's why I created this component